pkg/network/tracer/tracer_linux_test.go

```diff
@@ -2154,3 +2155,35 @@ func (s *TracerSuite) TestOffsetGuessIPv6DisabledCentOS() {
 	// fail if tracer cannot start
 	_ = setupTracer(t, cfg)
 }
+
+func (s *TracerSuite) TestPESisDisabledForUnsupportedKernels() {
+	t := s.T()
+	cfg := testConfig()
+	cfg.EnableProcessEventMonitoring = true
+	kv, err := ebpfkernel.NewKernelVersion()
+	kv4150 := kernel.VersionCode(4, 15, 0)
+	require.NoError(t, err)
+	if kv.Code >= kv4150 || kv.IsRH8Kernel() {
+		t.Skip("This test should only be run on kernels < 4.15.0 and RH7")
+	}
+	// fail if tracer cannot start
+	tr := setupTracer(t, cfg)
```

Comment:
That we re-write tr.cfg is a bit of an implementation detail. Can we instead check if tr.events is running or something less likely to change in the future.